Source/FieldSolver/SpectralSolver/SpectralBinomialFilter.H

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2020 David Grote
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef WARPX_SPECTRAL_BINOMIAL_FILTER_H_
+#define WARPX_SPECTRAL_BINOMIAL_FILTER_H_
+
+#include "SpectralHankelTransform/HankelTransform.H"
+#include "SpectralKSpace.H"
+
+/**
+ * \brief Class that sets up binomial filtering in k space
+ *
+ * Contains the filter arrays for r and z.
+ */
+class SpectralBinomialFilter
+{
+    public:
+
+        // `KFilterArray` holds a one 1D array ("ManagedVector") that
+        // implements the filter.
+        using KFilterArray = amrex::Gpu::ManagedVector<amrex::Real>;
+
+        SpectralBinomialFilter() {};
+        void InitFilterArrayR(HankelTransform::RealVector const & kr,
+                              amrex::Real const dr,
+                              int const npasses,
+                              bool const compensation);
+        void InitFilterArrayZ(RealKVector const & kz,
+                              amrex::Real const dz,
+                              int const npasses,
+                              bool const compensation);
+
+        KFilterArray const & getFilterArrayR() {return filter_r;};
+        KFilterArray const & getFilterArrayZ() {return filter_z;};
```

Comment:
I would add a whitespace between function names and parentheses, as these are not function calls:
```suggestion
        SpectralBinomialFilter () {};
        void InitFilterArrayR (HankelTransform::RealVector const & kr,
                               amrex::Real const dr,
                               int const npasses,
                               bool const compensation);
        void InitFilterArrayZ (RealKVector const & kz,
                               amrex::Real const dz,
                               int const npasses,
                               bool const compensation);

        KFilterArray const & getFilterArrayR () {return filter_r;};
        KFilterArray const & getFilterArrayZ () {return filter_z;};
```

Reply:
This is a stylistic difference. I've been adding the space between the function name and parenthesis only in the actual implementation of the function, usually in the cpp file. But I don't have a strong preference here so I'll accept this suggestion.